docs/node/run-your-node/prerequisites/cloud-providers.md

```diff
@@ -0,0 +1,31 @@
+# Cloud Providers
+
+Before committing to a service be sure to verify the processor compatibility and enquire with the provider about the status of Intel SGX support. Intel maintains a comprehensive list of processors that support Intel SGX:
+
+ * https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions-processors.html
+
+
+## Possible Limitations
+
+While many bare-metal or dedicated server providers use Intel processors that support SGX, there are potential limitations:
+
+ 1. **BIOS Configuration:** Some providers may not allow customers to access or modify BIOS settings, which is necessary to enable SGX.
```

Comment:
Enabling SGX is insufficient, one has to also set other BIOS settings mentioned in https://docs.oasis.io/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee#bios-configuration.

Reply:
https://www.alibabacloud.com/help/en/ecs/user-guide/build-an-sgx-encrypted-computing-environment

Alibaba officially support Intel SGX, I haven't tested them but I assume that given they have lots of documentation around using SGX then SGX would work.

Reply:
Have changed to:

> Some providers may not allow customers to access or modify BIOS settings, which may be necessary to correctly configure Intel SGX.

With the assumption that some providers may have specific SGX instance types which have all the necessary BIOS changes, whereas  others you will have to set one or more BIOS options to correctly configure Intel SGX.